configs/BOTWINGF722/config.h

```diff
@@ -0,0 +1,137 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU                       STM32F7X2
+#define BOARD_NAME                          BOTWINGF722
+#define MANUFACTURER_ID                     BLDY
+
+// ADC
+#define ADC_CURR_PIN                        PC0
+#define ADC_VBAT_PIN                        PC2
+#define ADC1_DMA_OPT                        0
+#define ADC_INSTANCE                        ADC1
+#define DEFAULT_CURRENT_METER_SOURCE        CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE        VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE         400
+#define DEFAULT_VOLTAGE_METER_SCALE         110
+
+// BARO
+#define USE_BARO
+#define USE_BARO_DPS310
+#define BARO_I2C_INSTANCE                   I2CDEV_3
+
+// BEEPER
+#define BEEPER_INVERTED
+#define BEEPER_PIN                          PB6
+
+// DEBUG
+#define SWDIO_PIN                           PA13
+#define SWCLK_PIN                           PA14
+
+// FLASH
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define SPI1_NSS_PIN                        PA4
+#define SPI1_SCK_PIN                        PA5
+#define SPI1_SDI_PIN                        PA6
+#define SPI1_SDO_PIN                        PA7
+#define FLASH_CS_PIN                        SPI1_NSS_PIN
+#define FLASH_SPI_INSTANCE                  SPI1
+#define DEFAULT_BLACKBOX_DEVICE             BLACKBOX_DEVICE_FLASH
+
+// I2C
+#define I2C1_SCL_PIN                        PB8
+#define I2C1_SDA_PIN                        PB9
+#define I2C3_SCL_PIN                        PA8
+#define I2C3_SDA_PIN                        PC9
+
+// IMU
+#define USE_ACC
+#define USE_GYRO
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_ICM42688P
+#define SPI3_NSS_PIN                        PD2
+#define SPI3_SCK_PIN                        PC10
+#define SPI3_SDI_PIN                        PC11
+#define SPI3_SDO_PIN                        PC12
+#define GYRO_1_EXTI_PIN                     PC13
+#define GYRO_1_CS_PIN                       SPI3_NSS_PIN
+#define GYRO_1_SPI_INSTANCE                 SPI3
+
+// LED
+#define LED_STRIP_PIN                       PA0
+#define LED0_PIN                            PC4
+#define LED1_PIN                            PB2
+
+// MAG
+#define MAG_I2C_INSTANCE                    I2CDEV_1
+
+// MOTOR
+#define MOTOR1_PIN                          PB5
+#define MOTOR2_PIN                          PB4
+#define MOTOR3_PIN                          PB3
+#define MOTOR4_PIN                          PA15
+#define MOTOR5_PIN                          PB0
+#define MOTOR6_PIN                          PB1
+#define MOTOR7_PIN                          PB10
+#define MOTOR8_PIN                          PB11
```

Comment:
Delete M5-8?

Reply:
This PR was committed one day before the new policy got in place. 

@sugaarK @blckmn